sonaranalyzer-dotnet/its/expected/Automapper/AutoMapper-025B9F93-7953-454B-AF29-E6A55B2AA58F-S4136.json

```diff
@@ -2,6 +2,30 @@
 "issues":  [
 {
 "id":  "S4136",
+"message":  "All 'AutoMapperConfigurationException' method overloads should be adjacent.",
```

Comment:
These new issues in the IT actually look a bit suspicious. I think it's a new kind of FP